app/controllers/resources_controller.rb

```diff
@@ -4,13 +4,24 @@ class ResourcesController < ApplicationController
   before_action :authorize_request
 
   # POST /objects
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/MethodLength
   def create
     begin
       response_cocina_obj = Dor::Services::Client.objects.register(params: cocina_model)
     rescue Dor::Services::Client::UnexpectedResponse => e
       return render build_error('Bad Request', e, '400', :bad_request)
     rescue Dor::Services::Client::ConnectionFailed => e
-      return render build_error('Unable to reach dor-services-app', e, '504', :gateway_timeout)
+      return render build_error('Unable to reach dor-services-app', e, '504',
+                                :gateway_timeout)
+    end
+
+    # Doing this here rather than IngestJob so that have accurate timestamp for milestone.
+    begin
+      workflow_client.create_workflow_by_name(response_cocina_obj.externalIdentifier, 'registrationWF', version: 1)
```

Comment:
Should this be in dor-services-app in the create method?  Then this service won't need a dependency on the workflow service.

Reply:
It turns out that argo doesn't apply registrationWF to all objects.  We can remove this line once https://github.com/sul-dlss/dor-services-app/issues/681 is complete.